9c-internal/all-in-one/application.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: argoproj.io/v1alpha1
 kind: Application
 metadata:
-  name: 9c-network
+  name: 9c-internal
```

Comment:
out `9c-internal-v2` cluster is using `9c-network` namespace now. so if you dont want to change namespace, it shouldnt be updated